api/webview/urls.py

```diff
@@ -7,5 +7,6 @@
     url(r'^documents/status', views.status, name='status'),
     url(r'^documents/(?P<source>\w+)/$', views.DocumentsFromSource.as_view(), name='source'),
     url(r'^documents/(?P<source>[a-z]+)/(?P<docID>(.*))/$', views.document_detail, name='document_detail'),
+    url(r'^institutions/', views.institution_detail),
```

Comment:
The name is used to reference it elsewhere, like from a template with the url tag - {% url name_goes_here %} so you don't neeeeed one if you're not going to reference it from a django template. But if you are, maybe, might as well give it a name